package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "sailthru-client",
     "description": "Node.js client for Sailthru API",
-    "version": "1.0.8",
+    "version": "2.0.0",
     "author": {
         "name": "Prajwal Tuladhar",
```

Comment:
@georgekliao what do we want to do with all these client libraries that have Praj's contact info?